docs/tutorials/sending-and-receiving-messages-with-mqtt.md

```diff
@@ -109,6 +115,10 @@ test it on our PC or on another EV3:
 Note: the second EV3 (the "Subscriber") just needs the "paho-mqtt" library,
 there is no need to install the "mosquitto" daemon.
 
+Note: when the publisher sends a string as payload use 'decode()' as in the
+example above When the Publisher sends a number, you can use 'int(msg.payload)'
```

Comment:
This formatting/wording seems a bit odd -- a capitalized "When" in the middle of the sentence doesn't make the intent clear.